examples/hybrid/gravitywave_parameterization/gravity_wave_parameterization.jl

```diff
@@ -0,0 +1,228 @@
+function gravity_wave_cache(
+    ::Type{FT};
+    source_height = FT(15000),
+    Bm = FT(1.2),
+    F_S0 = FT(4e-3),
+    dc = FT(0.6),
+    cmax = FT(99.6),
+    c0 = FT(0),
+    kwv = FT(2π/100e5),
+    cw = FT(40.0),
```

Comment:
Should we document these? or are these eventually moving into a struct / extracted from clima parameters?

Reply:
They should not go into clima parameters, but I do not know where is the best place to save this so for now this setup follows how the vertical diffusion saves its parameters...